views/user.py

```diff
@@ -0,0 +1,102 @@
+from flask import request
+from flask_restful import Resource
+from models import User
+from http.client import BAD_REQUEST
+from http.client import CREATED
+from http.client import INTERNAL_SERVER_ERROR
+from http.client import NO_CONTENT
+from http.client import NOT_FOUND
+from http.client import OK
+
+
+def non_empty_str(val, name):
+    """ Custom type for reqparser, blocking empty strings. """
+    if not str(val).strip():
+        raise ValueError('The argument {} is not empty'.format(name))
+    return str(val)
+
+
+def email_exists(email):
+    """
+    Check that an email exists in the User table.
+    returns: bool
+    """
+    user = User.select().where(User.email == email)
```

Comment:
non posso poi usare `exists()`, in quanto mi dice che `AttributeError: 'User' object has no attribute 'exists'`.
Per usare `User.get()` dovrei fare un try..except per l'eccezione _`models.UserDoesNotExist`: Instance matching query does not exist_ e tornare False... Se non ci sono problemi prestazionali preferirei lasciare la select ed usare `exists`, dimmi tu.